mrtt-ui/src/components/SiteInterventions.js

```diff
@@ -265,6 +369,143 @@ function SiteInterventionsForm() {
           <ErrorText>{errors.biophysicalInterventionsUsed?.message}</ErrorText>
         </FormQuestionDiv>
 
+        {displayMangroveSpeciesUsed() ? (
+          <FormQuestionDiv>
+            <StickyFormLabel>{questions.mangroveSpeciesUsed.question}</StickyFormLabel>
+            <List>
+              {mangroveSpeciesForCountriesSelected.length ? (
+                mangroveSpeciesForCountriesSelected.map((specie, specieSelectedIndex) => (
```

Comment:
this map has a lot of logic. It could be a named thing to improve readability in a perfect world.